scripts/cosm/deploy_erc20.js

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env node
+
+/* eslint-disable @typescript-eslint/camelcase */
+const { Encoding } = require("@iov/encoding");
+const {
+  encodeSecp256k1Signature,
+  makeSignBytes,
+  marshalTx,
+  logs,
+  RestClient,
+  Secp256k1Pen,
+} = require("@cosmwasm/sdk");
+const fs = require("fs");
+
+const httpUrl = "http://localhost:1317";
+const networkId = "testing";
+const defaultFee = {
+  amount: [
+    {
+      amount: "5000",
+      denom: "ucosm",
+    },
+  ],
+  gas: "1000000", // one million
+};
+const faucetMnemonic =
+  "economy stock theory fatal elder harbor betray wasp final emotion task crumble siren bottom lizard educate guess current outdoor pair theory focus wife stone";
+const faucetAddress = "cosmos1pkptre7fdkl6gfrzlesjjvhxhlc3r4gmmk8rs6";
+
+async function uploadContract(client, pen, wasm) {
+  const memo = "My first contract on chain";
+  const storeCodeMsg = {
+    type: "wasm/store-code",
+    value: {
+      sender: faucetAddress,
+      wasm_byte_code: Encoding.toBase64(wasm),
+      source: "https://github.com/confio/cosmwasm/raw/0.7/lib/vm/testdata/contract_0.6.wasm",
```

Comment:
This is not the proper source. Better leave it blank, or take it as an argument.

(It confused me until I saw line 81)

Reply:
Yeah, I removed the hardcoded values now to avoid blowing up the script